crates/task/src/lib.rs

```diff
@@ -151,6 +151,8 @@ pub enum VariableName {
     File,
     /// A path of the currently opened file (relative to worktree root).
     RelativeFile,
+    /// A path of the currently opened file's directory (relative to worktree root).
+    RelativeDir,
```

Comment:
Let's keep this consistent with `VariableName::Dirname`
```suggestion
    RelativeDirname,
```